apps/adminer.sh

```diff
@@ -99,7 +104,7 @@ The script will exit."
     exit 1
 else
 msg_box "Adminer was sucessfully installed and can be reached here:
-https://$ADDRESS:8443
+https://$ADDRESS:9443
```

Comment:
I would additionally add the port to NONO_PORTS

Reply:
Yes, just wanted to make sure it worked first.

8443 wasn't in nono ports so that's why I used it in the first place. ;)